src/core/public/doc_links/doc_links_service.ts

```diff
@@ -57,6 +57,9 @@ export class DocLinksService {
         cloud: {
           indexManagement: `${ELASTIC_WEBSITE_URL}/guide/en/cloud/current/ec-configure-index-management.html`,
         },
+        console: {
+          guide: `${ELASTIC_WEBSITE_URL}guide/en/kibana/current/console-kibana.html`,
```

Comment:
I think it would be better to use `${KIBANA_DOCS}` because it already has the correct version specified.